contracts/lib/GatewayLib.sol

```diff
@@ -29,47 +29,18 @@ library GatewayLib {
 
     /* Constants */
 
-    bytes32 constant STAKE_INTENT_TYPEHASH = keccak256(
+        bytes32 constant public STAKE_INTENT_TYPEHASH = keccak256(
         abi.encode(
             "StakeIntent(uint256 amount,address beneficiary,address gateway)"
         )
     );
 
-    bytes32 constant REDEEM_INTENT_TYPEHASH = keccak256(
+    bytes32 constant public REDEEM_INTENT_TYPEHASH = keccak256(
```

Comment:
What's the reason to make both the constants public?

Reply:
Adding public creates a getter for these constants.